### PR TITLE
Improving Drag and Drop

### LINKED ISSUE #1446: Improve Block drag & drop
- [ ] Show a grey bar when the user can drop (after dragging)
- [ ] Make droppable area full width
- [ ] Evaluate moving to beautiful DnD

Edit_Plone_Site (1)

### PR DESCRIPTION
**Pull request by iFlameing**
Fixes #1446

### CODE REVIEW

**Comment by sneridagh** [comment]
@iFlameing I've polished it a bit. I will continue this weekend a bit too.

In the meanwhile, can you take a look at the droppable styling?. I've found this:
https://codesandbox.io/embed/github/Utkarshbhimte/beatiful-dnd-custom-placeholder/tree/master/?fontsize=14

https://medium.com/@larryliu_50901/how-to-finally-add-a-custom-placeholder-react-beautiful-dnd-1b1359a94e30

coming from:
https://github.com/atlassian/react-beautiful-dnd/issues/518

Also fixing the tests would be great!

**Comment by sneridagh** [comment]
@tiberiuichim @avoinea @pnicolli @nzambello This is a nice one with great UX improvement, however, it does change the wrapper structure in the blocks editor (for good). 

I left untouched the internal (existing) class names and structure, hoping that will make the transition easier for existing customizations.

It's my intention to release Volto 6 soon (this weekend), and given the nature of this one, I expect to be able to include it in this release.

What do you think?

File: src/components/manage/Form/Form.jsx [hunk 1]
```diff
@@ -223,6 +227,17 @@ class Form extends Component {
...
   }
 
+  hideHandler = data => {
+    return (
+      data['@type'] === 'text' &&
+      (!data.text ||
+        (data.text &&
+          data.text.blocks &&
+          data.text.blocks.length === 1 &&
+          data.text.blocks[0].text === ''))
```
Comment (pnicolli) [inline src/components/manage/Form/Form.jsx:237]:
```suggestion
        (data.text?.blocks?.length === 1 &&
          data.text.blocks[0].text === ''))
```

**Comment by sneridagh** [comment]
@pnicolli Tried it out in my current local project and works like a charm :)

I will prepare it to be merged, I will wait until tomorrow, so you all can test it, then merge it and release Volto 6 :)

**Pull request merged**